xbmc/addons/Skin.cpp

```diff
@@ -369,6 +369,9 @@ const INFO::CSkinVariableString* CSkinInfo::CreateSkinVariable(const std::string
 
 void CSkinInfo::OnPreInstall()
 {
+  bool skinLoaded = g_SkinInfo != nullptr;
```

Comment:
Well previous patch had endless loop when upgrading the skin from v14 to v17 so isnt that the case here?